### ISSUE TITLE
Question about step time graph in Overview page

### ISSUE THREAD

**Issue by shradhasehgal** [pytorch/kineto#774]
Hi, I am wondering what 'step' on the X axis represents in the step-time graph on the overview page. 
I set my profiling schedule with 5 steps for 'active', yet the profiling results only include time for step 0 only and not steps 0 - 4. 
Could you clarify what 'step' here refers to if not each of the step numbers the profiler was 'active' for?

<img width="1343" alt="Screenshot 2023-06-27 at 6 19 45 PM" src="https://github.com/pytorch/kineto/assets/13078034/28f47356-9c6f-42ed-9ecc-1f6e1ed79513">

**Comment by shradhasehgal** [comment]
These are the environment details:

```
torch==2.0.1
torch-tb-profiler==0.4.1
```

Code:
```import os
import torch
import torch.nn as nn
import torch.backends.cudnn as cudnn
import torch.optim
import torch.utils.data
import torchvision
import torchvision.transforms as T
import torchvision.models as models
import time

model = models.resnet50(pretrained=True)

transform = T.Compose([T.Resize(256), T.CenterCrop(224), T.ToTensor()])
trainset = torchvision.datasets.CIFAR10(root='./data', train=True,
                                        download=True, transform=transform)
train_loader = torch.utils.data.DataLoader(trainset, batch_size=32,
                                          shuffle=True, num_workers=0)

criterion = nn.CrossEntropyLoss().cuda()
optimizer = torch.optim.SGD(model.parameters(), lr=0.001, momentum=0.9)
device = torch.device("cpu")
model.train()

def train(data):
    inputs, labels = data[0].to(device=device), data[1].to(device=device)
    outputs = model(inputs)
    loss = criterion(outputs, labels)
    optimizer.zero_grad()
    loss.backward()
    optimizer.step()

with torch.profiler.profile(
        schedule=torch.profiler.schedule(wait=1, warmup=1, active=100, repeat=1),
        on_trace_ready=torch.profiler.tensorboard_trace_handler('./test/resnet18'),
        record_shapes=True,
        profile_memory=True,
        with_stack=True
) as prof:
    for step, batch_data in enumerate(train_loader):
        if step >= (1 + 1 + 100):
            break
        train(batch_data)
        prof.step()
```

**Comment by anupambhatnagar** [comment]
Step refers to the iteration. Each step is determined by the `prof.step()` call in the code. The tensorboard plugin is not maintained so it might be showing something erroneous. Collecting a trace for 100 steps will result in a huge file. I suggest trying the following `schedule(skip_first=5, wait=5, warmup=2, active=2, repeat=1)`. You can also derive some insights of the collected trace using [Holistic Trace Analysis](https://hta.readthedocs.io/en/latest/index.html).

**Comment by shradhasehgal** [comment]
@anupambhatnagar Thanks for your reply. I reduced the number of 'active' iterations to 5, but still see the same error of just step 0 and not step 0 to 4. 
May be a bug in the tensorboard plugin - should I raise this issue elsewhere?

**Comment by anupambhatnagar** [comment]
Can you check if the trace has 5 steps? You can open the trace in `chrome://tracing`.

**Comment by shradhasehgal** [comment]
I verified there are indeed 5 steps in the trace using chrome://tracing. 
I can also see the steps various in the other graphs in tensorboard. Only the step time graph in the overview page has this issue.

**Comment by aamijar** [comment]
I had the same issue but what worked for me is to install torch-tb-profiler from source.
There is a new version `0.4.3` that is not available via pip but you can install from source by cloning the repo and following readme instructions https://github.com/pytorch/kineto/tree/main/tb_plugin

**Comment by aaronenyeshi** [comment]
Thanks for the workaround, closing this as tb_plugin is deprecated. Installing from source is a valid solution for those using it.
